public/cli/lib/commands/convert.js

```diff
@@ -195,7 +196,20 @@ Command.run = function(options) {
  function prepareLogger( done, numLogsToKeep=10 ) {
 
     // make sure the logging directory exists
-    var pathToLogDir = path.join(__dirname, "..", "..", "..", "..", "logs");
+    var pathToLogDir;
+    switch(process.platform) {
+        case "darwin":
+            pathToLogDir = path.join(os.homedir(), "Library", "Logs", "bbk", "logs");
+            break;
+
+        case "win32":
+            pathToLogDir = path.join(os.homedir(), "AppData", "Roaming", "bbk", "logs");
```

Comment:
Change `bbk` folder to `bible-karaoke`. In Windows (at least) there is already a folder for it when the app is installed.